ocs_ci/utility/utils.py

```diff
@@ -2313,7 +2314,8 @@ def ceph_health_check_base(namespace=None):
     """
     namespace = namespace or config.ENV_DATA["cluster_namespace"]
     health = run_ceph_health_cmd(namespace)
-
+    if type(health) is tuple:
```

Comment:
Dont' like this to be honest as you are not returning string as we were suppose to return from run_ceph_health_cmd.

I think this function should handle to run the health check two times on both namespaces to check status of both: internal/external mode.

Reply:
I disagree. I don't want to handle every time the function is called both health checks as when I install the external I want to know external health and when internal - intenal health. What I can do is to return a string when I do ssh and drop this check.
Like change this https://github.com/red-hat-storage/ocs-ci/blob/3ff20a16395ddfe08a61214523fc099f9cabca2f/ocs_ci/utility/utils.py#L2369
to:
```
ssh_output = connection_to_cephcluster.exec_cmd("ceph health")
return ssh_output[1]
```
After this I can remove the tuple check.

Reply:
What I meant is the first stage I'm installing internal mode and then I would like to check that internal is healthy and then install the external and check that the external is healthy. We can have a new function that call both. WDYT?

Reply:
https://github.com/shyRozen/ocs-ci/blob/892b7caeb562ee873e91d2a5b5f87535045e727e/ocs_ci/utility/utils.py#L2328